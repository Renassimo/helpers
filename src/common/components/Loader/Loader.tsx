import Box from '@mui/material/Box';
import LinearProgress from '@mui/material/LinearProgress';

const Loader = () => (
  <Box width="100%" position="fixed">
    <LinearProgress />
  </Box>
);

export default Loader;
