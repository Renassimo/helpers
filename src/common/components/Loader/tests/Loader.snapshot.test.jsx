import renderWithTheme from '@/tests/helpers';

import Loader from '../Loader';

describe('Loader snapshot', () => {
  test('renders successfully', () => {
    // Arrange
    // Act
    const { container } = renderWithTheme(<Loader />);
    // Assert
    expect(container).toMatchSnapshot();
  });
});
