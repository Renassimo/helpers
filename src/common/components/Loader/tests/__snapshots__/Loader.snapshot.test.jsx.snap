// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader snapshot renders successfully 1`] = `
<div>
  <div
    class="MuiBox-root css-sy48ec"
  >
    <span
      class="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate css-eglki6-MuiLinearProgress-root"
      role="progressbar"
    >
      <span
        class="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Indeterminate css-5ir5xx-MuiLinearProgress-bar1"
      />
      <span
        class="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar2Indeterminate css-1r8wrcl-MuiLinearProgress-bar2"
      />
    </span>
  </div>
</div>
`;
